### ISSUE TITLE
Can't get cfn-lint to work with coc

### ISSUE THREAD

**Issue by glnds** [joenye/coc-cfn-lint#1]
I can't get this plugin working. Other linting works fine. Cfn-lint on the cmd line also runs fine...
Not sure if there's a conflict with https://github.com/neoclide/coc-yaml
How can i tell to only lint certain files with coc-cfn-lint? like cfn-*.yml and cnf-*.yaml?

**Comment by joenye** [comment]
Two issues here:

1. There is an [issue](https://github.com/redhat-developer/vscode-yaml/issues/177) with [coc-yaml](https://github.com/neoclide/coc-yaml) where it gives lint errors in YAML CloudFormation templates. I've tried a few workarounds and the only one that worked reliably for me is disabling linting in all YAML files:

    ```yaml
    "yaml.validate": "false"
    ```

    Alternatively, you can keep linting enabled for all YAML files, and ignore _most_ errors in CloudFormation templates by setting `yaml.customTags`. See [here](https://github.com/redhat-developer/vscode-yaml/issues/177#issuecomment-588151203) for an example of `yaml.customTags`.

2. This plugin should lint alongside coc-yaml, even if linting is enabled in both. This plugin only lints a file if it detects the file as being valid CloudFormation. You can run `:CocInfo` inside vim to see the coc-cfn-lint logs and you'll see whether your file is being detected as CloudFormation. It currently doesn't do this check by looking at filename. Instead, it looks at file contents. By default, it will detect the file as CloudFormation if it contains _both_ "AWSTemplateFormatVersion" and "Resources". In my case, I'm working with non-standard template files that don't always contain _both_ these lines, so I have set the `cfnlint.detectCfnRegExps` as follows:

    ```yaml
    "cfnlint.detectCfnRegExps": ["AWS::", "\\n?\"Resources\"?\\s*:"],
    ```

    These RegExps ensure the plugin will lint any file containing "AWS::" _or_ "Resources:". If your file isn't being detected as valid CloudFormation, then consider adding your own RegExps here.

**Comment by glnds** [comment]
cfn-lint works fine now but i had to disable yaml linting.
```
"yaml.validate": "false"
```
Setting custom tags is not sufficient as a solution, the CloudFormation syntax still keeps giving false positives with those set.
